go.mod

```
@@ -30,6 +30,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/splunkhecexporter v0.107.1-0.20240816183440-44ef4d6c5a2c
 	github.com/open-telemetry/opentelemetry-collector-contrib/extension/ackextension v0.107.1-0.20240816183440-44ef4d6c5a2c
 	github.com/open-telemetry/opentelemetry-collector-contrib/extension/basicauthextension v0.107.1-0.20240816183440-44ef4d6c5a2c
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/headerssetterextension v0.107.0
```

```suggestion
	github.com/open-telemetry/opentelemetry-collector-contrib/extension/headerssetterextension v0.107.1-0.20240816183440-44ef4d6c5a2c
```
as we seem to be working off a 107 snapshot.